README.md

```diff
@@ -136,7 +138,7 @@ use of the library. Please refer to the COPYING file.
 poliastro comes from Polimi, which is the shortened name of the
 Politecnico di Milano, the Italian university where I was studying while
 writing this software. It\'s my tiny tribute to a place I came to love.
-*Grazie mille!*
+_Grazie mille!_
```

Comment:
Also, this can be reverted back ... GitHub works correctly with normal Markdown syntax :)

Reply:
Same here !!